tools/gulptasks/api-tree.js

```diff
@@ -64,18 +64,36 @@ async function apiTree() {
 
     const sources = argv.sources;
 
-    const moduleFiles = FSLib
-        .getFilePaths((sources || Path.join('ts', 'Core')), true)
-        .filter(path => !(
-            path.endsWith('Options.d.ts') ||
-            path.endsWith('Options.ts')
-        ));
-    const optionFiles = FSLib
-        .getFilePaths((sources || 'ts'), true)
-        .filter(path => (
-            path.endsWith('Options.d.ts') ||
-            path.endsWith('Options.ts')
-        ));
+    let singleFile = false;
+    let moduleSource = Path.join('ts', 'Core');
+    let optionSource = 'ts';
```

Comment:
Should we follow the syntax with variables in new lines with commas?

Reply:
Good question. The comma approach was back then enforced on Highcharts code, but not on tooling. I personally prefer separated statements over the list, but you could escalate it and I would fix it. 😅

Reply:
I'm fine with separated statements 😁